src/Elastic.Apm/BackendComm/CentralConfig/RuntimeConfigurationSnapshot.cs

```diff
@@ -117,12 +117,17 @@ internal RuntimeConfigurationSnapshot(IConfigurationReader mainConfiguration, Ce
 		public IReadOnlyList<WildcardMatcher> TransactionIgnoreUrls =>
 			_dynamicConfiguration?.TransactionIgnoreUrls ?? _mainConfiguration.TransactionIgnoreUrls;
 
+		public IReadOnlyCollection<WildcardMatcher> TransactionNameGroups =>
+			_mainConfiguration.TransactionNameGroups;
```

Comment:
Does it make sense to discuss making this dynamic with the rest of the Agents?

Reply:
Potentially, yes. Will add to the weekly agenda.

Reply:
Discussed and we'll make this dynamic. I'll update this PR.